litex/tools/litex_json2dts_linux.py

```diff
@@ -624,17 +633,27 @@ def add_clkout(clkout_nr, clk_f, clk_p, clk_dn, clk_dd, clk_margin, clk_margin_e
     return dts
 
 def main():
+    initrd_enabled = True
+
     parser = argparse.ArgumentParser(description="LiteX's CSR JSON to Linux DTS generator")
     parser.add_argument("csr_json", help="CSR JSON file")
     parser.add_argument("--initrd-start", type=int,            help="Location of initrd in RAM (relative, default depends on CPU)")
     parser.add_argument("--initrd-size",  type=int,            help="Size of initrd (default=8MB)")
```

Comment:
https://docs.python.org/3/library/argparse.html#:~:text=same%20interface.%20The-,BooleanOptionalAction,-is%20available%20in
> You may also specify an arbitrary action by passing an Action subclass or other object that implements the same interface. The BooleanOptionalAction is available in argparse and adds support for boolean actions such as --foo and --no-foo:
> ```
> >>>
> >>> import argparse
> >>> parser = argparse.ArgumentParser()
> >>> parser.add_argument('--foo', action=argparse.BooleanOptionalAction)
> >>> parser.parse_args(['--no-foo'])
> Namespace(foo=False)
> ```

Reply:
Right, I saw that was available, bit I went with this implementation add it's what I found in existing litex code.  I'll change it though.